patch/preconfig/series

```diff
@@ -1 +1 @@
-changelog.patch
+# changelog.patch
```

Comment:
Will you remove this line and this file?

Reply:
not necessary